engine/physics/src/physics/physics_2d.cpp

```diff
@@ -31,6 +31,8 @@ namespace dmPhysics
     : m_Worlds()
     , m_DebugCallbacks()
     , m_Gravity(0.0f, -10.0f)
+    , m_PositionIteration(16)
+    , m_VelocityIteration(16)
```

Comment:
Note that for backwards compatibility, these are not the current values.
In physics_2d.cpp, we pass in values 10,10